Kitodo/src/main/java/org/kitodo/services/schema/SchemaService.java

```diff
@@ -494,6 +494,7 @@ private void addMissingMetadata(DocStruct inTopStruct, Prefs prefs, Process proc
         Metadata mdPPNBand = null;
         Metadata mdSorting = null;
         try {
+            //TODO: why is it created here and next not assigned?
```

Comment:
Is this TODO for the `mdTitle`? If so and  `mdTitle` is not used yet, then if could be removed.